README.md

```diff
@@ -77,6 +80,7 @@ toast(options, cb);
 *   `type` - Can be one of `info`, `success`, `warning`, or `error`
 *   `icon` - Override the default icon
 *   `time` - Duration to keep the toast open, 0 to wait until closed by the user
+*    `onClick` - The function that will be called when you click on the toast
```

Comment:
`onClose` is also a toast option :)

Reply:
We are getting somewhere! Lets just add this `onClose` option to the docs and I will merge it :)